src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -6,7 +6,7 @@ import StartFinishTimeInput from '../start-finish-time-input/StartFinishTimeInpu
 import dayjs from 'dayjs';
 import utc from 'dayjs/plugin/utc';
 
-const EditShiftHours = ({ setShowEditShiftHours }) => {
+const EditShiftHours = ({ setShowEditShiftHours, timeEntry, index }) => {
```

Comment:
`index` variable naming - same comment as in editShiftTimecard, not completely obvious what it's referring to.